shiny/experimental/api-examples/card/app.py

```diff
@@ -15,6 +15,7 @@
         ui.p("This is still the body."),
         x.ui.card_footer("This is the footer"),
         full_screen=True,
+        id="card1",
```

Comment:
Because this ID is only really for testing, we should prolly copy it into an e2e app instead.  The e2e app will have the `id=` and the api-examples will not contain an `id=`.

-------------------------

This goes for the value_box app as well.